<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sjabloon</class>
 <widget class="QFrame" name="Sjabloon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>71</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>441</width>
    <height>71</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>441</width>
    <height>71</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Shape::StyledPanel</enum>
  </property>
  <widget class="QLineEdit" name="lesuur">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
   <property name="placeholderText">
    <string>Uur</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>21</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="onderwerpen">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>12</number>
   </property>
   <property name="placeholderText">
    <string>Onderwerp(en)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locaties">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>12</number>
   </property>
   <property name="placeholderText">
    <string>Locatie(s)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="verwijderButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Verwijder</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="startTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="endTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>21</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="naam">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Naam</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>startTime</tabstop>
  <tabstop>endTime</tabstop>
  <tabstop>onderwerpen</tabstop>
  <tabstop>locaties</tabstop>
  <tabstop>lesuur</tabstop>
  <tabstop>naam</tabstop>
  <tabstop>verwijderButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
